spec/tbd_tests_spec.rb

```diff
@@ -2217,7 +2218,7 @@
 
     # Now mimic (again) the export functionality of the measure
     out2 = JSON.pretty_generate(io2)
-    outP2 = File.dirname(__FILE__) + "/../json/tbd_warehouse2.out.json"
+    outP2 = File.join(__dir__, "../json/tbd_warehouse2.out.json")
```

Comment:
@macumber: Being more systematic for file read/write.

Reply:
Using join is a good idea